src/site/layouts/q_a.drupal.liquid

```diff
@@ -10,10 +10,12 @@
         <div class="usa-content">
           <!-- Search bar -->
           <div class="medium-screen:vads-u-border-bottom--2px vads-u-border-color--gray-light medium-screen:vads-u-margin-bottom--3">
-            {% include "src/site/includes/support_resources_search_bar.drupal.liquid" %}
+            <div class="vads-u-padding-x--1 large-screen:vads-u-padding-x--0">
+              {% include "src/site/includes/support_resources_search_bar.drupal.liquid" %}
+            </div>
           </div>
```

Comment:
This wrapper aligns the horizontal width of the search bar

![image](https://user-images.githubusercontent.com/1915775/109022780-28fa7880-768a-11eb-8d74-43a7dc253c68.png)